openforcefield/__init__.py

```diff
@@ -1,13 +1,3 @@
-try:
-    import openeye
-except Exception as e:
-    print(e)
-    print('Warning: Cannot import openeye toolkit; not all functionality will be available.')
```

Comment:
Agree that removing this should be fine. something in `ToolkitWrapper` creates a warning about this anyway, so two "missing OpenEye" messages were being printed before this change.